journal/modernized/37-chapter-34.adoc

```diff
@@ -634,6 +634,7 @@ the very scum and offscouring of the nations which gave them birth,
 and not a few of these my own countrymen: however, under all these unhappy circumstances,
 the New Zealanders are a noble race of men;
 and though far behind-hand in that gentleness and docility which
+//"behind-hand" was circled in the book. Not sure if it should be changed, or what to.
```

Comment:
yeah, that's tough. let's change it to just `behind` in the modernized, and leave it in the original. 👍